web/src/api/openapi-schema/forbiddenResponse.ts

```
@@ -0,0 +1,14 @@
+ * Do not edit manually.
+ * storyden
+ * Storyden social API for building community driven platforms.
+The Storyden API does not adhere to semantic versioning but instead applies a rolling strategy with deprecations and minimal breaking changes. This has been done mainly for a simpler development process and it may be changed to a more fixed versioning strategy in the future. Ultimately, the primary way Storyden tracks versions is dates, there are no set release tags currently.
+
+ * OpenAPI spec version: rolling
+ */
+
+/**
+ * Forbidden
+ */
+export type ForbiddenResponse = void;
```

Biome (1.9.4)

[error] void is confusing outside a return type or a type parameter.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)